secant/MockedDependencies/KeyStoring.swift

```diff
@@ -7,32 +7,27 @@
 
 import Foundation
 
+struct StoredWallet: Codable, Equatable {
+    var birthday: BlockHeight?
+    let seedPhrase: String
+    let version: Int
+}
+
 protocol KeyStoring {
```

Comment:
Now that we have defined the desired behavior, we should add documentation to this methods

Reply:
done